Apromore-Custom-Plugins/CSVImporter-Logic/src/main/java/org/apromore/service/csvimporter/services/legacy/LogImporterCSVImpl.java

```diff
@@ -42,26 +42,26 @@
 import org.deckfour.xes.model.impl.XAttributeTimestampImpl;
 import org.springframework.stereotype.Service;
 
+import javax.inject.Inject;
 import java.io.*;
```

Comment:
If you're alphabetizing the imports in this block, "javax.inject.Inject" comes alphabetically after "java.io.*".